full-stack-tests/core/src/frontend/hub/IModelConnection.test.ts

```diff
@@ -114,7 +114,7 @@ describe("IModelConnection (#integration)", () => {
   });
 
   // TODO: This test currently causes other tests to fail due to how it restarts IModelHost
-  it.skip("should be able to re-establish IModelConnection if the backend is shut down", async () => {
+  it("should be able to re-establish IModelConnection if the backend is shut down", async () => {
```

Comment:
@calebmshafer is this worth attempting to salvage?

Reply:
Not quite sure if this is really worth testing in this way... It's really validating that there's no issue with sharing state  between backend and frontend by restarting the backend.

I'd say remove it.